crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -62,6 +63,7 @@ void awslc_fips_service_indicator_update_state(enum fips_approved_algorithm_t se
   indicator->service_id = service_id;
   if(indicator->counter == UINT64_MAX) {
     OPENSSL_PUT_ERROR(EVP, ERR_R_OVERFLOW);
+    indicator->counter = 0;
     return;
   }
```

Comment:
I don't know if we need the check and the error. I think we can just wrap around quietly. Is that bad?

Reply:
I removed the error output. I don't think it makes sense to output an error message, if we've already wrapped around quietly ourselves either.